src/components/PeopleFilters.tsx

```diff
@@ -1,23 +1,91 @@
-export const PeopleFilters = () => {
+import { FC, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { SearchLink } from './SearchLink';
+import cn from 'classnames';
+
+const centuriesList = [16, 17, 18, 19, 20];
+
+interface Props {
+  setPeopleFilter: React.Dispatch<React.SetStateAction<Person[]>>;
+  people: Person[];
+}
+
+export const PeopleFilters: FC<Props> = ({ setPeopleFilter, people }) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
+
+  function handleQueryChange(event: React.ChangeEvent<HTMLInputElement>) {
+    const params = new URLSearchParams(searchParams);
+
+    params.set('query', event.target.value);
+    setSearchParams(params);
+  }
+
+  function toggleCentery(century: number) {
+    const newCenturies = centuries.includes(century.toString())
+      ? centuries.filter((c: string) => c !== century.toString())
```

Comment:
move century.toString to variable
```suggestion
    const newCenturies = centuries.includes(century.toString())
      ? centuries.filter((currentCentury: string) => currentCentury !== century.toString())
```